pkg/cri/server/restart.go

```diff
@@ -81,6 +82,24 @@ func (c *criService) recover(ctx context.Context) error {
 
 				return nil
 			}
+
+			// Convert the old pause Container to the new Sandbox
+			info, _ := sb.Container.Info(ctx)
```

Comment:
What if err is not nil?

Reply:
as the commit is reverted, will consider this in the next PR